doc/modules/mod_mam.md

```diff
@@ -79,6 +79,10 @@ These options will only have effect when the `odbc` backend is used:
  When set to `internal`, stores messages and JIDs in internal format.
  **Warning**: Archive MUST be empty to change this option.
 * **async_writer** (boolean, default: `true`) - Enables asynchronous writer that is faster than synchronous but harder to debug.
+  This async writer does not message routing, on the other hand messages are stored in the archive with a delay.
+* **flush_interval** (integer, default: `2000`) How often (in milliseconds) buffered message are flushed to db.
+* **max_buffer_size** (integer, default, `30`) Max size of the buffer for async writer.
```

Comment:
Or just `batch_size`.

Reply:
Right, `max_batch_size` is more meaningful. I'll change that.